service/src/main/java/org/ehrbase/dao/access/jooq/CompositionAccess.java

```diff
@@ -883,7 +883,11 @@ public String getLinks() {
 
   @Override
   public void setFeederAudit(FeederAudit feederAudit) {
-    compositionRecord.setFeederAudit(JSONB.valueOf(new FeederAuditEncoding().toDB(feederAudit)));
+    if (feederAudit == null) {
+      compositionRecord.setFeederAudit(null);
+    } else {
+      compositionRecord.setFeederAudit(JSONB.valueOf(new FeederAuditEncoding().toDB(feederAudit)));
+    }
   }
```

Comment:
Can you also refactor  all usage of
' JSONB.valueOf(new FeederAuditEncoding().toDB(composition.getFeederAudit())))'


to use this method.

Reply:
There is also a bug in line 194 -197

Reply:
> There is also a bug in line 194 -197

fixed.
also: should the links attribute also be updateable? because it currently is not..

Reply:
Also implemented for the links attribute